Sources/SwiftProtobuf/BinaryEncoder.swift

```diff
@@ -82,63 +80,41 @@ public struct BinaryEncoder {
         append(value ? 1 : 0)
     }
 
-    mutating func putFixedUInt64(value : UInt64) {
-        var v = value
+    mutating func putFixedUInt64(value: UInt64) {
+        var v = value.littleEndian
         let n = MemoryLayout<UInt64>.size
-        withUnsafePointer(to: &v) { v -> () in
-            v.withMemoryRebound(to: UInt8.self, capacity: n) { p -> () in
-                let buff = UnsafeBufferPointer<UInt8>(start: p, count: n)
-                append(contentsOf: buff)
-            }
-        }
+        memcpy(pointer, &v, n)
```

Comment:
I experimented a bit with different ways to put a fixed-size number into the serialized output.  Using `memcpy` like this is the best I've found so far.